src/pages/Admin/index.tsx

```diff
@@ -11,24 +11,22 @@ import {
 } from './style';
 import { Container } from '../Sign/style';
 import { useEffect, useState } from 'react';
+import { useQuery } from '@tanstack/react-query';
 import axiosInstance from '@/utils/customAxios';
 import { Button, Input, Text } from '@/components';
-import { Message } from '@/types/APIResponseTypes';
 import { useDispatch } from '@/store';
 import { getMyInfo } from '@/slices/user/thunk';
+import { Message } from '@/apis/responseModel';
+import { Spinner } from '@/components/MainPageSpinner/style';
 
 const Admin = () => {
-  const [messages, setMessages] = useState([]);
   const [channelName, setChannelName] = useState('');
   const dispatch = useDispatch();
 
-  useEffect(() => {
-    const getMessages = async () => {
-      const { data } = await axiosInstance.get('messages/conversations');
-      setMessages(data);
-    };
-    getMessages();
-  }, []);
+  const { data: messages, isLoading } = useQuery({
+    queryKey: ['request'],
+    queryFn: async () => await axiosInstance.get('messages/conversations'),
```

Comment:
```suggestion
    queryFn: async () => await axiosInstance.get<Message[]>('messages/conversations'),
```
이렇게 주는것이 선언적으로 any 로 추론되는것을 막을수 있을것 같다는생각이드는데 어떻게 생각하실까요?!

Reply:
그게 좋겠군요